### DISCUSSION TITLE
Liveries

### DISCUSSION THREAD

**Discussion by E-Savageair** [Sadia2000/747-400#49] [General]
Hello,
I saw your paintkit that features top and bottom. for the 747-400  So I'm asking if I can be permitted to paint Qantas' "Wunala Dreaming" scheme since this was not possible before with the older paintkit.  Other wise I can wait....

**Comment by naviat-dev** [comment]
Feel free to do whatever you want with the paintkit. If your livery is good enough, I can add it to the repo.

**Comment by Sadia2000** [comment]
This project is a voluntary task, so if you feel like helping, feel free!

**Comment by E-Savageair** [comment]
Thank you. I'll get started.

    On Friday, September 16, 2022 at 06:26:26 PM CDT, Israel Emmanuel ***@***.***> wrote:  
 
 


Feel free to do whatever you want with the paintkit. If your livery is good enough, I can add it to the repo.
